docs/en/relying-party-solution.rst

```diff
@@ -1,21 +1,742 @@
 
```

Comment:
mmhh no,

this was moved in the file remote-flow.rst, therefore it must not be copied in this file